[12.0][ADD] account_multicurrency_revaluation_rate_type: create module (#106)
OCA/account-closing: account_multicurrency_revaluation_rate_type/readme/DESCRIPTION.rst

> This addon add possibility to choose in which currency type reevaluation would be

Reviewer: ```suggestion
This addon adds the possibility to choose which currency rate type will be used for revaluation.
```

> made. It extends account_multicurrency_revaluation behaviour to use month currency rates

Reviewer: ```suggestion
It extends account_multicurrency_revaluation behaviour to use monthly currency rates
```
